arca/backend/base.py

```diff
@@ -138,15 +133,13 @@ def run(self, repo: str, branch: str, task: Task, git_repo: Repo, repo_path: Pat
         """
         python_path = self.get_or_create_environment(repo, branch, git_repo, repo_path)
 
-        script_name, script = self.create_script(task)
-        script_path = Path(self._arca.base_dir, "scripts", script_name)
-        script_path.parent.mkdir(parents=True, exist_ok=True)
-        script_path.write_text(script)
+        task_filename, task_json = self.serialized_task(task)
 
-        logger.info("Stored task script at %s", script_path)
+        task_path = Path(self._arca.base_dir, "tasks", task_filename)
+        task_path.parent.mkdir(parents=True, exist_ok=True)
+        task_path.write_text(task_json)
 
-        st = os.stat(str(script_path))
-        script_path.chmod(st.st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
+        logger.info("Stored task definition at %s", task_path)
```

Comment:
This is out of scope of this PR, but maybe you should stick to one formatting style (i.e. f-strings, not %).

Reply:
I've been doing this because of logging lazy evaluation (https://stackoverflow.com/a/4149231) - I thought it was standard practice? I don't work with logging that often to know... If it actually is not the standard practice I'll change (in the future)...

Reply:
Oh, I didn't know about his. OK, keep it. Sorry about that.